src/explorer.ts

```diff
@@ -416,23 +424,25 @@ export class KubernetesFileObject implements KubernetesObject {
 
 class HelmReleaseResource implements KubernetesObject {
     readonly id: string;
+    readonly status: string;
```

Comment:
This doesn't appear to be used?

Reply:
This is used at: `treeItem.iconPath = getIconForHelmRelease(this.helmRelease.status.toLowerCase());`

Reply:
Fixed